CMakeLists.txt

```diff
@@ -228,8 +239,17 @@ if(NOT WOLFSSL_SINGLE_THREADED)
     endif()
 endif()
 
-# TODO: - DTLS
-#       - TLS v1.3 Draft 18
+# TODO: - TLS v1.3 Draft 18
```

Comment:
Please remove `# TODO: - TLS v1.3 Draft 18`. The draft support was removed.

Reply:
Removed.